_includes/prod_deployment/secure-generate-certificates.md

```diff
@@ -1,3 +1,5 @@
+You can use either `cockroach cert` commands or [`openssl` commands](create-security-certificates-openssl.html) to generate the security certificates. This section discusses the procedure to generate certificates using the `cockroach cert` commands.
```

Comment:
Can tighten this up a bit:

In the first sentence: `generate the security certificates` > `generate security certificates`

The second sentence can be cut to: `This section features the `cockroach cert` commands.`